dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -146,9 +150,27 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
                   ))}
                 </select>
               </div>
-              {this.props.enableBasicForm ? this.renderBasicForm() : this.renderAdvancedForm()}
+              <div className="margin-t-big" hidden={!this.props.enableBasicForm}>
```

Comment:
I'm a bit confused at this point - it looks like we're no longer using "enableBasicForm" as a feature flag... Shouldn't the snapshot below be unchanged since it's rendered without the feature flag set to true?

I expected this to do something like:
```
{this.props.enableBasicForm ? this.renderBasicAdvancedFormTabs() : this.renderAdvancedForm() }
```
Not sure if I missed it.

Reply:
it seems it's not really readable so I will rewrite this. Yes, that's how it works but instead of using the conditional here, I am using the `hidden` property. Then, below is where I am rendering the basic or the advanced form depending on the state (that btw, `showBasicForm` will always be `false` if `enableBasicForm` is `false` as well).